rust/nasl-interpreter/src/built_in_functions/array.rs

```diff
@@ -83,4 +144,119 @@ mod tests {
         assert_eq!(parser.next(), Some(Ok(make_dict!())));
         assert_eq!(parser.next(), Some(Ok(make_dict!())));
     }
+
+    #[test]
+    fn make_list() {
+        let code = r###"
+        a = [2,4]
+        make_list(1, 0);
+        make_list();
+        make_list(1,NULL,2);
+        b = make_array("el", 6);
+        make_list(1, 0, b);
+        make_list(1, 0, a);
+        "###;
+        let storage = DefaultSink::new(false);
+        let mut register = Register::default();
+        let loader = NoOpLoader::default();
+        let mut interpreter = Interpreter::new("1", &storage, &loader, &mut register);
+        let mut parser =
+            parse(code).map(|x| interpreter.resolve(&x.expect("no parse error expected")));
+        assert_eq!(parser.next(),
+                   Some(Ok(NaslValue::Array(vec![NaslValue::Number(2),NaslValue::Number(4)]))));
+        assert_eq!(parser.next(),
+                   Some(Ok(NaslValue::Array(vec![NaslValue::Number(1),NaslValue::Number(0)]))));
+        assert_eq!(parser.next(), Some(Ok(NaslValue::Array([].into()))));
+        assert_eq!(parser.next(),
+                   Some(Ok(NaslValue::Array(vec![NaslValue::Number(1),NaslValue::Number(2)]))));
+        parser.next();
+        assert_eq!(parser.next(),
+                   Some(Ok(NaslValue::Array(vec![
+                       NaslValue::Number(1),
+                       NaslValue::Number(0),
+                       NaslValue::Number(6)]))));
+        assert_eq!(parser.next(),
+                   Some(Ok(NaslValue::Array(vec![
+                       NaslValue::Number(1),
+                       NaslValue::Number(0),
+                       NaslValue::Number(2),
+                       NaslValue::Number(4)]))));
+    }
+
+    #[test]
+    fn sort() {
+        let code = r###"
+        a = make_array(5, 6, 7, 8);
+        l = make_list("abbb", 1, "aaaa", 0, a);
+        s = sort(l);
+        "###;
+        let storage = DefaultSink::new(false);
+        let mut register = Register::default();
+        let loader = NoOpLoader::default();
+        let mut interpreter = Interpreter::new("1", &storage, &loader, &mut register);
+        let mut parser =
+            parse(code).map(|x| interpreter.resolve(&x.expect("no parse error expected")));
+        parser.next();
+        parser.next();
+        let a = parser.next();
+        let b = Some(Ok(NaslValue::Array(vec![
+                       NaslValue::Number(0),
+                       NaslValue::Number(1),
+                       NaslValue::Number(6),
+                       NaslValue::Number(8),
+                       NaslValue::String("aaaa".to_string()),
+                       NaslValue::String("abbb".to_string())])));
+        assert_eq!(a, b);
+        println!("Sort {:?}", a);       
+        println!("Sort {:?}", b);       
```

Comment:
I guess this can be removed?

Reply:
It doesn't hurt. It won't be printed  unless you run the tests as following (which I founded useful for debugging)
`cargo test -- --nocapture`
If you want me to remove it, no issue.